TheMovieDb/Components/CircularProgressBarView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CircularProgressBarView.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 30/10/21.
+//
+
+import UIKit
+
+protocol CircularProgressBarViewP {
+  func progressAnimation(duration: TimeInterval)
+  func createCircularPath()
+}
+
+final class CircularProgressBarView: UIView, CircularProgressBarViewP {
+  
+  private var circleLayer = CAShapeLayer()
+  private var progressLayer = CAShapeLayer()
+  private var startPoint = CGFloat(-Double.pi / 2)
+  private var endPoint = CGFloat(3 * Double.pi / 2)
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+  }
+  
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+  }
+  
+  func createCircularPath() {
```

Comment:
Couldn't see where are we using this func. if we are not using outside the class, we can make it private or remove it if is not used